combat-trainer.lic

```diff
@@ -1655,6 +1664,7 @@ class SpellProcess
                    .select { |spell| spell['min_threshold'] ? game_state.npcs.length >= spell['min_threshold'] : true }
                    .select { |spell| spell['max_threshold'] ? game_state.npcs.length <= spell['max_threshold'] : true }
                    .select { |spell| spell['expire'] ? Flags["ct-#{spell['abbrev']}"] : true }
+                   .select { |spell| !game_state.is_offense_allowed? ? spell['harmless'] : true }
```

Comment:
Thanks for accepting my suggestion on line 1453, I believe the same can be applied here, too?
```suggestion
                   .select { |spell| game_state.is_offense_allowed? || spell['harmless'] }
```